engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -362,6 +362,72 @@ enum SemanticsRole {
 
   /// The main display for a tab.
   tabPanel,
+
+  /// An input field for users to enter search terms.
+  searchBox,
+
+  /// A control that used for dragging across content.
+  ///
+  /// e.g, the drag handle of [ReorderableList].
+  dragHandle,
+
+  /// A control to cycle through content on tap.
+  ///
+  /// e.g, the next and previous month button of a [CalendarDatePicker].
+  spinButton,
+
+  /// A input field with a dropdown list box attached.
+  ///
+  /// e.g, a [DropDownMenu]
+  comboBox,
+
+  /// Contains a list of [menu].
+  ///
+  /// e.g, a [MenuBar].
+  menuBar,
+
+  /// A button that opens a dropdown that contains multiple [menuItem]s.
+  ///
+  /// e.g, a [MenuAnchor] or [DropDownButton].
+  menu,
+
+  /// Items in a dropdown created by [menu] or [comboBox].
+  menuItem,
+
+  /// A container to display multiple [listItem] in vertical or horizontal
+  /// layout.
+  ///
+  /// e.g, a [LisView] or [Column].
+  list,
+
+  /// An item in a [list].
+  listItem,
+
+  /// An area that resembles a form.
+  form,
+
+  /// A pop up when hover over a component to provide contextual explanation.
+  tooltip,
+
+  /// A control that controls the scrolling of content within a viewing
+  /// area.
+  scrollBar,
+
+  /// A graphic object that spins to indicate the application is busy.
+  ///
+  /// e.g, a [CircularProgressIndicator].
+  loadingSpinner,
+
+  /// A graphic object that shows progress along a line.
```

Comment:
Does it have to be "along a line"? This role could be used for a progress indicator designed like this:

![Screenshot 2025-02-11 at 3 31 18 PM](https://github.com/user-attachments/assets/61f90a4d-f21c-4eb6-86a8-a4bd66e27897)

Reply:
This is specific for https://api.flutter.dev/flutter/material/LinearProgressIndicator-class.html

does this circular progress indicator consider a progress'bar'?

Reply:
Good question. Do the visuals matter for screen reader purposes? If it represents some progress from 0% to 100%, it is some sort of progress bar-ish thing.

Reply:
updated the doc to not restrict to linear bar